sale_commission_advance/models/account_move.py

```diff
@@ -0,0 +1,84 @@
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    is_downpayment = fields.Boolean(default=False, copy=False)
```

Comment:
This field is conflicting: on one part, the generic name can collapse with another and it's only a helper one. On the other, it won't be set for down payments done prior to the module installation.

You should deduce if something is a down payment other way.